CONTRIBUTING.md

```diff
@@ -41,75 +25,97 @@ process documented here.
 
 All contributors must agree to the project [code of conduct](CODE_OF_CONDUCT.md).
 
-## Golang Coding Style
+## Pull requests
+
+All the repositories accept contributions by a GitHub Pull request (PR).
```

Comment:
I wonder if we want to hotlink out to github for a PR reference? https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-requests

Reply:
Good call. Thanks.